tests/integration/telemetry/api/dashboard_test.go

```diff
@@ -306,7 +306,9 @@ func setupDashboardTest(done <-chan struct{}) {
 			times++
 			scopes.Framework.Infof("sending traffic %v", times)
 			for _, ing := range ingr {
-				host, port := ing.TCPAddress()
+				hosts, ports := ing.TCPAddresses()
```

Comment:
This code is only used to fill the dashboard with data.
No need to run using both addresses